outlines/fsm/parsing.py

```diff
@@ -614,6 +652,8 @@ def __init__(self, conf: "LexerConf", states, always_accept=()):
                 lexer_conf.terminals = [
                     terminals_by_name[n] for n in accepts if n in terminals_by_name
                 ]
+                if not lexer_conf.terminals:
+                    continue
```

Comment:
Note: Bug-fix for case where no `lexer_conf.terminals` is empty (happens when EOS is only legal next terminal)